### DISCUSSION TITLE
Whether any time delay of 5.0.0 can be merged into the 4.9.x branch？

### DISCUSSION THREAD

**Discussion by onejimmyboy** [apache/rocketmq#6277] [Ideas]
The current 5.0 version supports any length of delay, is it considered to be supported in the 4.9.x version? How feasible is it? Is there such a case？

**Comment by onejimmyboy** [comment]
还有一个问题，如果代码合入了4.9.x版本，spring boot框架在发送消息时，是不是可以使用rocketmq-spring-boot-starter最新版本：2.2.3

**Reply by echooymxq** [reply]
为什么不考虑直接升级到5.0呢？

**Reply by onejimmyboy** [reply]
因为我们一直使用的是4.9.x版本，考虑升级会带来一些风险，并且5.0.0刚迭代了两个版本，如果使用5.0.版本可能后期还需要更新小版本。目前我们只需要支持任意时长延迟，打算5.0再迭代几个小版本后再升上去

**Reply by drpmma** [reply]
There is no timeline for now.

**Reply by drpmma** [reply]
Most of the new features added in RocketMQ 5.0 are implemented through new modules, such as proxy and ha, without affecting the existing functions. Additionally, RocketMQ 5.0 includes many optimizations and bug fixes. Even if the new functions such as proxy or ha are not needed, it is still possible to continue using RocketMQ 5.0 in the same way as RocketMQ 4.9 without any functional dependencies.
